common/src/main/java/com/microsoft/identity/common/internal/broker/AppData.java

```diff
@@ -44,74 +44,75 @@
 /**
  * Represents packageName and SignatureHash of a broker app.
  */
-public class BrokerData {
+public class AppData {
```

Comment:
Does this class really need to be `public`? If yes, do we want to consider taking it out of the `internal` package space?